ocs_ci/deployment/aws.py

```diff
@@ -22,6 +22,11 @@
 from ocs_ci.utility.retry import retry
 from ocs_ci.utility.utils import run_cmd, clone_repo
 from .deployment import Deployment
+from tests import helpers
```

Comment:
We shouldn't be importing anything from tests. If something exists in the `helpers` module that is needed outside of the `tests` module, we should move it to a framework module instead. This will stop us from running into issues if we decide to relocate our tests to another repository.

Reply:
Done, removed it